clio-server/src/main/scala/org/broadinstitute/clio/server/service/ReadGroupService.scala

```diff
@@ -49,23 +79,40 @@ object ReadGroupService {
   }
 
   private[service] val ConverterV1Key =
-    SameFieldsTypeConverter[TransferReadGroupV1Key, ModelReadGroupKey]
+    CaseClassTypeConverter[TransferReadGroupV1Key, ModelReadGroupKey] {
+      _.updated("location", "Unknown")
+    }
 
   private[service] val ConverterV1Metadata =
     SameFieldsTypeConverter[
       TransferReadGroupV1Metadata,
       ModelReadGroupMetadata
     ]
 
+  private[service] val ConverterV2Key =
+    SameFieldsTypeConverter[TransferReadGroupV2Key, ModelReadGroupKey]
+
   private[service] val ConverterV1QueryInput =
-    SameFieldsTypeConverter[
+    CaseClassTypeConverter[
       TransferReadGroupV1QueryInput,
       ModelReadGroupQueryInput
-    ]
+    ] { _.filterKeys(_ != "location") }
 
   private[service] val ConverterV1QueryOutput =
-    SameFieldsTypeConverter[
+    CaseClassTypeConverter[
       ModelReadGroupQueryOutput,
       TransferReadGroupV1QueryOutput
+    ] { _.filterKeys(_ != "location") }
```

Comment:
I think that's what you've written: "allow any key that is not 'location'". As the `TRGV1Output` doesn't have a `location` field, this makes sense.

However, the `TRGV1Input` does _not_ have a `location`. So I don't believe that needs to be filtered for `ConverterV1QueryInput`.